docs/store/filecoin-plus.md

```diff
@@ -37,16 +37,36 @@ Clients can use their DataCap to incentivize miners to serve their needs. This c
 
 ### Get DataCap
 
-Clients are required to have an on-chain Filecoin address where DataCap can be received. If you are setting up a new address, make sure to initialize it by sending a minimal amount of FIL to it (purchasing some FIL from an exchange, for example).
+Clients are required to have an on-chain Filecoin address where DataCap can be received. If you are setting up a new address, make sure to initialize it by sending a minimal amount of FIL to it (purchasing some FIL from an exchange, for example). You will need an address to proceed with getting DataCap in any of the following ways.
 
 _Note: As of network version 12, DataCap allocations are a single-use credit on a Filecoin address. If you receive an allocation and require more, you should make a new request with a unique address that you have initialized like above. [FIP-0012](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0012.md) was accepted and implemented in network version 13(actor v5), which allows client addresses to receive DataCap multiple times._
 
-Clients get DataCap by making a request to a notary.
+Clients get DataCap by making a request to a notary. For your first DataCap allocation of 32GiB, you can use an auto-verifier such as https://verify.glif.io/. Auto-verifiers exist to grant DataCap immediately to clients that can authenticate themselves via a specific method. For example, the verify.glif.io automatic notary grants DataCap to clients who have a GitHub account that is > 180 days old and has not been used at this site in the past 30 days.
+1. Head over to https://verify.glif.io/
+2. Connect your GitHub account - click the **Start** button on the top right of the page
+3. Sign in to GitHub if you have not already
+4. Paste in the address to which you'd like to receive DataCap in the box under "Request" and hit **Request**
+5. The auto-notary will now attempt to send a message to the Filecoin Network whereby your address will be granted 32GiB of DataCap. This will take about 1 min 
+6. When complete, you now have 32GiB to start making deals with! You can always come back to this site and use the "Check" box to see how much DataCap you have left on a specific address
+
+For receiving DataCap at a larger scale (for business needs and production use cases), depending on the amount of data being onboarded to the network, a client has two options.
+- Applying directly to a specific notary - best for clients looking for < 100TiB of DataCap
+- Applying for a Large Dataset notary dedicated to a specific project - best for clients looking for > 100TiB of DataCap (usually in the 500TiB-5PiB range)
+
+
+For applying directly to a specific notary:
```

Comment:
```suggestion
Clients get DataCap by making a request to a notary. For your first DataCap allocation of 32GiB, you can use an auto-verifier such as https://verify.glif.io/. Auto-verifiers exist to grant DataCap immediately to clients that can authenticate themselves via a specific method. For example, the verify.glif.io automatic notary grants DataCap to clients who have a GitHub account that is > 180 days old and has not been used at this site in the past 30 days.

1. Head over to https://verify.glif.io/
2. Connect your GitHub account - click the **Start** button on the top right of the page
3. Sign in to GitHub if you have not already
4. Paste in the address to which you'd like to receive DataCap in the box under "Request" and hit **Request**
5. The auto-notary will now attempt to send a message to the Filecoin Network whereby your address will be granted 32GiB of DataCap. This will take about 1 min 
6. When complete, you now have 32GiB to start making deals with! You can always come back to this site and use the "Check" box to see how much DataCap you have left on a specific address

For receiving DataCap at a larger scale (for business needs and production use cases), depending on the amount of data being onboarded to the network, a client has two options:

- Applying directly to a specific notary - best for clients looking for < 100TiB of DataCap
- Applying for a Large Dataset notary dedicated to a specific project - best for clients looking for > 100TiB of DataCap (usually in the 500TiB-5PiB range)


For applying directly to a specific notary:
```